week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -73,6 +93,33 @@ function insertShoppingList(shoppingList) {
  */
 function insertBooks(books) {
   //Write your code in here
+  let content = document.querySelector("#content");
+  let headingEl = document.createElement('h1')
+  headingEl.textContent = 'Book List'
+  content.appendChild(headingEl)
+  let ulList = document.createElement('ul')
+  content.appendChild(ulList)
+  for(let i = 0 ; i < books.length ; i++){
+    let liList = document.createElement('li')
+    liList.className = 'li-style'
+     let pEl = document.createElement('p')
+    pEl.innerHTML = books[i].title + ' - ' + books[i].author
+     liList.appendChild(pEl)
+     ulList.appendChild(liList)
+     let imgEl = document.createElement('img')
+     imgEl.style.height = '300px'
+     liList.appendChild(imgEl)
+    if (books[i].alreadyRead == true){
+      liList.style.backgroundColor = 'green'
+    } else {
+      liList.style.backgroundColor = 'red'
+    }
+  }
+  let imgLinks = document.getElementsByTagName('img')
+  imgLinks[0].src = 'https://m.media-amazon.com/images/I/416Hql52NCL.jpg'
+  imgLinks[1].src ='https://images-na.ssl-images-amazon.com/images/I/41m1rQjm5tL._SX322_BO1,204,203,200_.jpg'
+  imgLinks[2].src ='https://images-na.ssl-images-amazon.com/images/I/418M2053aNL._SX258_BO1,204,203,200_.jpg'
+  
 }
 
 //
```

Comment:
Your work here is correct but the instruction required from you create Three favorites list of the books using the DOM method to create elements and so on. Try to do everything using the DOM creation elements and what it's needed.